src/InteractiveWindow/Editor/InteractiveWindowResources.resx

```diff
@@ -225,4 +225,7 @@
   <data name="WaitMessage" xml:space="preserve">
     <value>Generating Rich Text Format representation from selection.</value>
   </data>
+  <data name="Paste" xml:space="preserve">
+    <value>Paste</value>
```

Comment:
Note we are very, very close to the localization cutoff. Are you aware of the ramifications of that?